ERROR_RESOLUTION_SUMMARY.md

```
@@ -1,3 +1,9 @@
+/**
+ * Author: Gailad Chesa
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)